### PR TITLE
Polling location finder

### PR DESCRIPTION
**Pull request by dbrusky1**
* Changed JSON file to correctly reference data
* Added dropdown in Rider Box next to Destination for Polling locations by state name
* Loads page in new window

### CODE REVIEW

**Comment by sasjkia** [comment]
Thanks for doing these @dbrusky1 - super helpful for when people sign up! What are the conflicts?

**Comment by dbrusky1** [comment]
I am working on getting conflicts

**Comment by dbrusky1** [comment]
I have been having trouble resolving this but I did get a branched created on the master repo called polling-location-finder that has all the correct code.  Could you merge this branch into master?

**Comment by dbrusky1** [comment]
Sorry about that....got lost in a downward spiral of github.  Allset now.

File: dropdown.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ var votingDropdown = (function() {
 	$.getJSON("res/voting-details.json", function(data) {
 		var items = [];
 		$.each(data, function(key, val) {
+			console.log(key);
```
Comment (rickr) [inline dropdown.js]: Is this log required?

File: res/styles.css [hunk 2]
```diff
@@ -64,6 +64,7 @@ h5 {
 #pnlIntro {
     color: #fff;
 }
+<<<<<<< HEAD
```
Comment (rickr) [inline res/styles.css]: Looks like this is left over from a conflict resolution.

**Comment by sasjkia** [comment]
@rickr is this good to merge now?

**Comment by sasjkia** [comment]
Looks like there are still some conflicts...

**Comment by dbrusky1** [comment]
Im on it.  Sorry for the trouble

**Comment by dbrusky1** [comment]
I have conflicts from creating the two dropdowns of the same code that need to be corrected.  I will close this and pull request to fix vote-dropdown.  Then I will be able to pull request again on this dropdown.

**Pull request closed without merging**